pio-scripts/output_bins.py

```diff
@@ -19,7 +19,7 @@ def _create_dirs(dirs=["map", "release", "firmware"]):
         os.makedirs(os.path.join(OUTPUT_DIR, d), exist_ok=True)
 
 def create_release(source):
-    release_name = _get_cpp_define_value(env, "WLED_RELEASE_NAME")
+    release_name = _get_cpp_define_value(env, "WLED_RELEASE_NAME").replace("\\\"", "")
```

Comment:
The quote stripping needs to be inside the `if` below, in case it's not defined and `_get_cpp_define_value` returns None.